test/test-cases/README.md

```diff
@@ -3,6 +3,20 @@ This contains a hierarchical set of directories containing DASH test cases organ
 
 # Contents
 
-| Folder                                         | Description                                               |
-| ---------------------------------------------- | --------------------------------------------------------- |
-| [vnet2vnet](vnet2vnet/README.md)                  | DASH vnet2vnet Tests 
+| Folder | Description |
+| --- | --- |
+| functional | Tests to verify essential functionality using low-rate traffic (SAI PTF).
+| scale | Tests with hight-rate traffic and complex configuration to verify scaling real-world scenarios.
+
+## functional
+
+| Folder/File | Description |
+| --- | --- |
+| saidashvnet.py | VNET-to-VNET test cases
```

Comment:
Need to add `saidashvnet` to .wordlist.txt to pass spellcheck.

Reply:
There might be some more customization available, see https://github.com/rojopolis/spellcheck-github-actions

Reply:
I used backticks for the filenames. It works.